test/my_tests/hash_inside_html_1.dot

```diff
@@ -0,0 +1,6 @@
+// https://github.com/pydot/pydot/issues/235
+graph G {
+    b1 [label=<
+         We are the knights who say <BR/># Ni
+    >];
+}
```

Comment:
I think if we're going to test this, we should also test the use of HTML-like strings as an identifier, in addition to a label. We can do it in the same file, but I'd add a second test, something like...

```suggestion

    <<b>b2 #rules!</b>> [color=red];
}
```

Reply:
I'm really surprised to learn that that is valid DOT but I've added the test!